recipes/uhd/0002-python-Fix-internal-library-name-incl.-suffix-to-mat.patch

```diff
@@ -0,0 +1,49 @@
+From c81af5ebe7ff1988f5175ec8085d372d2ddc0876 Mon Sep 17 00:00:00 2001
+From: Ryan Volz <rvolz@mit.edu>
+Date: Fri, 1 Nov 2019 10:32:24 -0400
+Subject: [PATCH 2/4] python: Fix internal library name (incl. suffix) to match
+ filename.
+
+Instead of renaming the library file, this sets the suffix in CMake so
+that the filename turns out as desired and also linker references know
+the correct name.
+---
+ host/python/CMakeLists.txt | 14 +++++++-------
+ 1 file changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/host/python/CMakeLists.txt b/host/python/CMakeLists.txt
+index a19391d83..d45303a01 100644
+--- a/host/python/CMakeLists.txt
++++ b/host/python/CMakeLists.txt
+@@ -23,6 +23,12 @@ execute_process(
+ 
+ # Build pyuhd library
+ add_library(pyuhd SHARED pyuhd.cpp)
++# python expects extension modules with a particular suffix
++if(WIN32)
++    set_target_properties(pyuhd PROPERTIES PREFIX "lib" SUFFIX ".pyd")
++else()
++    set_target_properties(pyuhd PROPERTIES PREFIX "lib" SUFFIX ".so")
```

Comment:
FYI, you should use the EXT_SUFFIX value described in PEP 3149, but not necessary to get this PR merged. See https://github.com/symengine/symengine.py/commit/fc5965fa1b34cba4266cc7b096161c5fa81c09b2 for an example.

Reply:
Great, good to know! I'll incorporate that in the patch I've sent upstream.